support/cas-server-support-redis-ticket-registry/src/test/java/org/apereo/cas/ticket/registry/RedisServerTicketRegistryTests.java

```diff
@@ -135,6 +141,32 @@ class WithoutRedisModulesTests extends BaseRedisSentinelTicketRegistryTests {
 
     }
 
+    @Nested
+    @TestPropertySource(properties = {
+            "cas.ticket.registry.redis.queue-identifier=cas-node-100",
+            "cas.ticket.registry.redis.host=localhost",
+            "cas.ticket.registry.redis.port=6379",
+            "cas.ticket.registry.redis.enable-redis-search=false",
+            "cas.ticket.registry.redis.crypto.enabled=false",
+            "cas.ticket.crypto.enabled=true"
+    })
+    class WithEncryptedIdentifiersTests extends BaseRedisSentinelTicketRegistryTests {
+
+        @Override
+        @BeforeEach
+        public void initialize(final TestInfo info, final RepetitionInfo repetitionInfo) throws Throwable {
+            super.initialize(info, repetitionInfo);
+
+            val tgtFactory = (TicketGrantingTicketFactory) ticketFactory.get(TicketGrantingTicket.class);
+            this.ticketGrantingTicketId = tgtFactory.create(CoreAuthenticationTestUtils.getAuthentication(), null,
+                    TicketGrantingTicket.class).getId();
+
+            val transientFactory = (TransientSessionTicketFactory) ticketFactory.get(TransientSessionTicket.class);
+            this.transientSessionTicketId = transientFactory.create(null).getId();
```

Comment:
Yes. In fact, by default, the ticket identifiers used in tests are manually generated, they are not "real" CAS ones. It can be an UUID or TGT-i or `TICKET_GRANTING_TICKET_ID_GENERATOR.getNewTicketId(TicketGrantingTicket.PREFIX)`.
This comes from `BaseTicketRegistryTests`.

As we don't rely on factories, the setting: "cas.ticket.crypto.enabled=true" is never taken into account. This is why I explicitly use the factories here. I didn't want to change the `BaseTicketRegistryTests` not to change the logic for all tests of ticket registries.